_episodes/02-raster-plot.md

```diff
@@ -1,29 +1,25 @@
 ---
-layout: post
-title: "Raster 01: Plot Raster Data in R"
-date:   2015-10-28
+title: "Plot Raster Data in R"
 authors: [Leah A. Wasser, Megan A. Jones, Zack Brym, Kristina Riemer, Jason Williams, Jeff Hollister,  Mike Smorul]
 contributors: [ ]
 packagesLibraries: [raster, rgdal]
 dateCreated:  2015-10-23
-lastModified: 2016-03-15
+lastModified: 2017-09-07
 categories:  [self-paced-tutorial]
 tags: [R, raster, spatial-data-gis]
```

Comment:
some of this is NEON tag stuff... i'm happy to help remove it - if it's useful via another PR ... just let me know!